pkgs/development/python-modules/wheel/default.nix

```diff
@@ -15,9 +15,12 @@ buildPythonPackage rec {
   version = "0.33.6";
```

Comment:
should we also bump the version of wheel? This is a rather old version, or is that less important for the bootstrapping process?

Reply:
old? 0.33.6 is the latest version

Reply:
oh wait, i was thinking setuptools, xD. Sorry :)